Mathlib/LinearAlgebra/QuadraticForm/TensorProduct.lean

```diff
@@ -79,12 +86,19 @@ theorem baseChange_tmul (Q : QuadraticForm R M₂) (a : A) (m₂ : M₂) :
     Q.baseChange A (a ⊗ₜ m₂) = Q m₂ • (a * a) :=
   tensorDistrib_tmul _ _ _ _
 
-
 theorem associated_baseChange [Invertible (2 : A)] (Q : QuadraticForm R M₂)  :
     associated (R₁ := A) (Q.baseChange A) = (associated (R₁ := R) Q).baseChange A := by
   dsimp only [QuadraticForm.baseChange, BilinForm.baseChange]
   rw [associated_tmul (QuadraticForm.sq (R := A)) Q, associated_sq]
 
+theorem polarBilin_baseChange [Invertible (2 : A)] (Q : QuadraticForm R M₂)  :
+    polarBilin (Q.baseChange A) = (polarBilin Q).baseChange A := by
+  rw [QuadraticForm.baseChange, BilinForm.baseChange, polarBilin_tmul, BilinForm.tmul,
+    ←LinearMap.map_smul, smul_tmul', ←two_nsmul_associated R]
+  erw [associated_sq]
```

Comment:
How about adding something like:
```lean
lemma associatedHom_apply {R M S : Type*} [CommRing R]
    [AddCommGroup M] [Module R M] [CommRing S] [Algebra S R] [Invertible (2 : R)]
    (Q : QuadraticForm R M) :
    associatedHom S Q = associated Q := by
  rfl
```
just below the definition of `associated` and using it to avoid the `erw` (so that this whole proof just becomes one giant chain of `rw`s).

Reply:
This `associatedHom` thing is a mess that I was considering cleaning up another time. I think the canonical spelling should be an unbundled one, with the hom available if needed.